src/ArenaReplay.cpp

```diff
@@ -311,248 +439,529 @@ class ReplayGossip : public CreatureScript
         player->PlayerTalkClass->ClearMenus();
         switch (action)
         {
-        case 1: // Replay 2v2 Matches
-            player->PlayerTalkClass->SendCloseGossip();
-            ShowLastReplays2v2(player, creature);
-            break;
-        case 2: // Replay 3v3 Matches
-            player->PlayerTalkClass->SendCloseGossip();
-            ShowLastReplays3v3(player, creature);
-            break;
-        case 3: // Replay 5v5 Matches
-            player->PlayerTalkClass->SendCloseGossip();
-            ShowLastReplays5v5(player, creature);
-            break;
-        case 4: // Saved Replays
-            player->PlayerTalkClass->SendCloseGossip();
-            ShowSavedReplays(player, creature);
-            break;
-        case GOSSIP_ACTION_INFO_DEF: // "Back"
-            OnGossipHello(player, creature);
-            break;
-
-        default:
-            if (action >= GOSSIP_ACTION_INFO_DEF + 10) // Replay selected arenas (intid >= 10)
-            {
-                return replayArenaMatch(player, action - (GOSSIP_ACTION_INFO_DEF + 10));
-            }
+            // Last 30 days:
+            case 1: // "Replay top 2v2 games of the last 30 days"
```

Comment:
instead of set `1` and write the comment next to it, we could use some constants like for `case GOSSIP_ACTION_INFO_DEF:`, we could create an `enum` to group all these numbers